### PR TITLE
misc-- oliver

### PR DESCRIPTION
**Pull request by o752d**
items added: 
  - (legacy) issues review
  - meeting notes for last
  - dir for doc references

### CODE REVIEW

**Review by yaahc — APPROVED** [review]
blessed, ty so much oliver.

File: meetings/2020-11-09.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+- Implement trait based proof of concept for `Backtrace` in `core` (Jane Lusby)
+- Update `object-provider` API in generic member access RFC (Jane Lusby)
+- Move forward with adding an Error impl for `&impl Error` (Ashley Mannix)
+  - Create an FCP on the existing PR
+  - Add guidance for using &dyn Error + 'static and impl Error + Send + Sync +
+    'static for borrow and owned errors respectively
+- Review the "Fix the error trait" tracking issue to figure out next steps
+  (Oliver + Ashley Mannix)
+- Review the panic plan rfc (Jane Lusby)
+- reach out to the author of the error codes issue (Jane Lusby)
```
Comment (epage) [inline meetings/2020-11-09.md:18]: @yaahc not seeing this in the past meeting notes or in this meetings minutes,  Which issue is this?

Comment (o752d) [reply meetings/2020-11-09.md:18]: You may find some information here: https://blog.rust-lang.org/inside-rust/2020/11/23/What-the-error-handling-project-group-is-working-on.html#one-standardized-error-trait

Comment (o752d) [reply meetings/2020-11-09.md:18]: @epage ping :)

Comment (yaahc) [reply meetings/2020-11-09.md:18]: https://github.com/rust-lang/project-error-handling/issues/7

Comment (yaahc) [reply meetings/2020-11-09.md:18]: https://github.com/rust-lang/project-error-handling/issues/7

**Pull request merged**